src/register-org/components/individual-registered-with-regulator/individual-registered-with-regulator.component.html

```diff
@@ -4,39 +4,34 @@
       <a class="govuk-back-link" href="javascript:void(0)" (click)="onBack()">Back</a>
       <xuilib-hmcts-error-summary [errorMessages]="validationErrors" [header]="'There is a problem'"></xuilib-hmcts-error-summary>
       <span class="govuk-caption-xl">Register an organisation</span>
+      <h1 class="govuk-heading-xl">
```

Comment:
@johnbenjamin-hmcts - same comment as above

Reply:
@Daniel-Lam - I have updated the code now.